pkgs/top-level/ocaml-packages.nix

```diff
@@ -226,6 +228,8 @@ let
 
         cmdliner_1_1 = callPackage ../development/ocaml-modules/cmdliner/1_1.nix { };
 
+        cbor = callPackage ../development/ocaml-modules/cbor { };
+
```

Comment:
Please keep this list alphabetically sorted.

Reply:
I thought I put it in the right place but I think it got unsorted after a rebase, fixing now